js/src/structured-data-blocks/how-to/utils/buildDurationString.js

```diff
@@ -0,0 +1,75 @@
+/* External dependencies */
+import { __, _n, sprintf } from "@wordpress/i18n";
+
+/**
+ * Tries to parse a string to an int and returns a default if it does not produce a valid number.
+ *
+ * @param {string} str          String to parse to an int.
+ * @param {number} [defaultInt] Default value if the parse does not return a valid number.
+ *
+ * @returns {number} The parsed number or default.
+ */
+function parseIntDefault( str, defaultInt = 0 ) {
+	return parseInt( str, 10 ) || defaultInt;
+}
+
+/**
+ * Transforms the durations into pluralized strings.
+ *
+ * @param {Object} durations         The duration values.
+ * @param {number} durations.days    Number of days.
+ * @param {number} durations.hours   Number of hours.
+ * @param {number} durations.minutes Number of minutes.
+ *
+ * @returns {Array<string>} Array of pluralized durations.
+ */
+function getPluralizedDurations( { days, hours, minutes } ) {
+	const plurals = [];
```

Comment:
These are not always plurals. If every value is 1, you'll have an array of singulars